static-site/app/pathogens/files/page.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Metadata } from "next";
+
+import FlexCenter from "../../../components/flex-center";
+import { FocusParagraphCentered } from "../../../components/focus-paragraph";
+import CorePathogenResourceListing from "./resources";
+import { SmallSpacer, HugeSpacer } from "../../../components/spacers";
+
+
+const title = "Workflow data files for core pathogens";
+
+export const metadata: Metadata = {
+  title,
+};
+
+export default function PathogenWorkflowFiles(): React.ReactElement {
+  return (
+    <div className="pageContent">
```

Comment:
I ended up undoing this, based on feedback on one of my PRs, so this can just be

```suggestion
    <div>
```